modules/rosa-creating-cluster.adoc

```diff
@@ -20,6 +20,31 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
+. Enable ROSA to create JSON files for account-wide roles and policies, including Operator policies. This is supported for the `rosa init account` mode only:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode manual --version 4.8 <1>
+----
+<1> Valid options for `--mode` are:
++
+* `auto`: Roles and policies will be created using the current AWS account. This is the default option.
+* `manual`: Policy documents will be saved in the current directory.
+
+. Enable ROSA to create cluster-specific roles. The roles created include the relevant prefix for the cluster name.
++
+[source,terminal]
+----
+$ rosa create operator-roles --mode manual --cluster <name|id>
+----
++
+This option takes in a parameter of the cluster name or ID and then outputs the relevant JSON file and sample command to create the OIDC provider, including the thumbprint:
++
+[source,terminal]
+----
+$ rosa create oidc-provider --mode manual --cluster <name|id>
+----
+
 . You can create a cluster using the default settings or by specifying custom settings using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
```

Comment:
This section on creating a cluster should be done after the account roles, but before the operator roles/oidc provider. The order of commands should be:
1. `rosa create account-roles`
2. `rosa create cluster`
3. `rosa create operator-roles`
4. `rosa create oidc-provider`